bot/exts/evergreen/realpython.py

```diff
@@ -0,0 +1,75 @@
+import logging
+from html import unescape
+from urllib.parse import quote_plus
+
+from discord import Embed
+from discord.ext import commands
+
+from bot import bot
+from bot.constants import Colours
+
+logger = logging.getLogger(__name__)
+
+
+API_ROOT = "https://realpython.com/search/api/v1/"
+ARTICLE_URL = "https://realpython.com{article_url}"
+SEARCH_URL = "https://realpython.com/search?q={user_search}"
+
+ERROR_EMBED = Embed(
```

Comment:
I think linters may complain about this?
```suggestion
SEARCH_URL = "https://realpython.com/search?q={user_search}"


ERROR_EMBED = Embed(
```